csrankings-a.csv

```diff
@@ -520,6 +520,7 @@ Albert Chern,Univ. of California - San Diego,https://page.math.tu-berlin.de/~che
 Albert Chi-shing Chung,HKUST,https://www.cse.ust.hk/~achung,MVf7Lq0AAAAJ
 Albert Cohen 0001,Ecole Normale Superieure,https://who.rocq.inria.fr/Albert.Cohen,MkKZKAMAAAAJ
 Albert Gatt,University of Malta,http://staff.um.edu.mt/albert.gatt,uF5HKZQAAAAJ
+Albert M. Lai,Washington University in St. Louis,https://informatics.wustl.edu/research-lab-albert-lai/,sxe41PcAAAAJ

```

Comment:
Dead link (https://i2db.wustl.edu/research-lab-albert-lai/).

Reply:
He has a courtesy appointment in CSE: https://cse.wustl.edu/faculty-research/courtesy-appointments.html. I decided not to add all the others because they don't have publications in any of the relevant papers.